site/src/pages/components/invokers.explainer.mdx

```diff
@@ -294,7 +294,7 @@ Invokers, the names and exact semantics may be subject to change:
 | `<details>`     | `'toggle'`            | If the `<details>` is `open`, then close it, otherwise open it                       |
```

Comment:
Did we decided if we were keeping auto values?

For example the dialog closedby attribute will have a missing value default of "auto" without it needing an associated keyword?